src/app/pages/login-page/login-form/login-form.component.scss

```diff
@@ -0,0 +1,54 @@
+$login-background: #f6f2fc;
+$login-border: #e9e1f5;
+
+.sign-form {
+  width: 360px;
+  p {
+    font-size: 1.7em;
+    text-align: center;
+  }
+  form {
+    border: 1px solid $login-border;
+    border-radius: 5px;
+    background-color: $login-background;
+    padding: 10% 7%;
+
+    mat-form-field {
+      display: block;
+      input {
+        padding: 3px;
+      }
+    }
+
+    button {
+      width: 100%;
+    }
+  }
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  justify-self: center;
+  height: 100%;
+  width: 100%;
+  .main {
+    height: 55%;
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+
+    justify-content: flex-end;
+    align-items: center;
+  }
+}
+
+::ng-deep .mat-form-field-flex > .mat-form-field-infix {
+  padding: 0.7em 0px !important;
+}
+
+@media only screen and (max-width: 380px) {
```

Comment:
I think we should find something in angular material or flex layout. Don't use media queries here.

Reply:
removed for this pr